autoptsclient_common.py

```diff
@@ -1294,6 +1298,37 @@ def run_recovery(args, ptses):
     except:
         traceback.print_exc()
 
+    try:
```

Comment:
Do we need this try/except here? If there are issues with a particular IXIT then we should handle it individually and let other IXIT values be set. Ideally, we would not cause exceptions in PTS, but just return an error from autopts server.

Reply:
No, we don't actually need it :) After reinit_pts above is called, setup_project_pixits(ptses) should work fine.